modules/src/music.py

```diff
@@ -10,9 +12,19 @@
 def process(input, entities):
     output = {}
     try:
+
+        auth_url = "https://accounts.spotify.com/api/token"
+        headers = {'Authorization': 'Basic ' + config.SPOTIFY_API_KEY}
+        payload = {'grant_type': 'client_credentials'}
+
+        r = requests.post(auth_url, headers=headers, data=payload)
+
+        key = r.json()['access_token']
```

Comment:
Better yet, we can save it in config.py to avoid needing to create any new files.